From b49bcb6eece322b70fc2e8b0c8498f5a38ea7b82 Mon Sep 17 00:00:00 2001
From: David McFarland <corngood@gmail.com>
Date: Sun, 26 Jul 2020 17:29:49 -0300
Subject: [PATCH] link radv with ld_args_build_id

This is needed for radv_device_get_cache_uuid to work correctly.
---
 src/amd/vulkan/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/amd/vulkan/meson.build b/src/amd/vulkan/meson.build
index eec026e8f60..a07a0821938 100644
--- a/src/amd/vulkan/meson.build
+++ b/src/amd/vulkan/meson.build
@@ -183,7 +183,7 @@ libvulkan_radeon = shared_library(
   ],
   c_args : [c_vis_args, no_override_init_args, radv_flags],
   cpp_args : [cpp_vis_args, radv_flags],
-  link_args : [ld_args_bsymbolic, ld_args_gc_sections, libvulkan_radeon_ld_args],
+  link_args : [ld_args_build_id, ld_args_bsymbolic, ld_args_gc_sections, libvulkan_radeon_ld_args],
   link_depends : [libvulkan_radeon_link_depends,],
   install : true,
 )
--
2.28.0
